pulumi/grapl/Pulumi.testing.yaml

```diff
@@ -4,15 +4,6 @@ config:
   grapl:cloudsmith-repository-name: grapl/testing
   grapl:confluent-environment-name: testing
   grapl:env_vars:
```

Comment:
these have been useless for quite some time; though i'll keep the mechanism in there i guess